### ISSUE TITLE
How to Fuse a changed animation with its original version

### ISSUE THREAD

**Issue by KamenMaskX** [0ceal0t/Dalamud-VFXEditor#44]
Hello there, sorry if this is too many words but I have been working with the editor and it has been great so far but I have reached a roadblock and am hoping to get some help figuring out how to do something I have seen on this video: https://youtu.be/5yZg3Dv9YWI?t=114

As you can see the animation looks as though they have managed to combine the Gekko animation with either the old Delirium DRK animation or it might be Jugular Rip from GNB. I wanted to do something similar to this with the Jinpu animation and combine it with Hypervelocity from GNB but I don't know how to.

The only method I can think of accomplishing this is to modify a different skill other than Jinpu to have the Jugular Rip animation and then stitch the 2 together but this presents an issue: all my other skills are either already modified or not really suited to modify, like buffs and such. Is there any other way to fuse the Jinpu and Jugular Rip animations that I am missing?

TL;DR: Trying to emulate an animation mod I saw in a YT video but I can't figure out how to do it without changing another animation I have already modded.

**Comment by 0ceal0t** [comment]
yeah, the two ways to accomplish this are either:
- move the jugular rip animation to some samurai .pap and trigger both of the animations with the .tmb
- export both the gekko and jugular rip .pap with tools like these (https://github.com/ilmheg/multiassist, https://github.com/0ceal0t/BlenderAssist) merge them, then re-export and replace the gekko .pap

**Comment by KamenMaskX** [comment]
Ok so those are really the only 2 ways then, at least I wasn't missing anything. The blender method seems quite complicated but I will give it a try because I don't have any abilities I am willing to mod to accomplish this besides maybe modding a buff or role skill just to have access to the animation. 

I have 3 more quick questions before closing this, I figure I may as well get them all out while I have the time:

1.  Would modifying the .pap of a skill samurai used to have just doesn't anymore work such as hissastu: seigan or ageha work for this purpose or would are those simply no longer loaded in as part of samurai?
2. A bit unrelated but while I am asking animation questions I may as well: at this point in the video: https://youtu.be/5yZg3Dv9YWI?t=133 they seem to be able to use the animation for kaeshi: namikiri but are somehow able to cut off the initial flourish from it, where you spin into the kneeling position. I assumed this was done by messing with the "duration" of the animation but have not been able to duplicate it, would you have any suggestions for that?
3. Is there a more efficient way to find sound effects to use than loading them up in the tmb editor? I thought godbert would help but that doesn't seem to show the path for sfx and ffxiv explorer doesn't seem to be the right tool for that either.

Thanks for the help, been really loving diving deep into the editor.

**Comment by 0ceal0t** [comment]
1. They probably aren't loaded anymore, but I'm not 100% sure
2. My first thought would be the duration either in the .tmb or .pap file. If that doesn't work, I'm not sure
3. Not really. I've got some stuff for sound files in the works, but nothing yet

**Comment by KamenMaskX** [comment]
Understood, thanks for all the help!
